lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -181,6 +181,12 @@ def execute(self, single_threaded: bool, delete: bool = True) -> int:
     prompt=False,
     help="Override the default controller name",
 )
+@click.option(
+    "--access",
+    required=True,
```

Comment:
Can you make this optional?  And then if specified, validate it is one of `public` or `private`?